<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>707</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="openKmlBtn">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>95</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>95</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>kml aç</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>325</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>323</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Noktalar Arası Mesafe (m)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="pointsRangeSpinBox">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999999999</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>273</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>273</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Oluşacak nokta sayısı</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="olusacakNoktaLE">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="analysisBtn">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>95</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>95</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Analiz Et</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="maxElevLBL">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>En yüksek kot = </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="minElevLBL">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>En düşük kot = </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalPastTimeLBL">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Toplam Geçen Süre = </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="avaragePastTimesLBL">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Her sorgu için ortalama geçen süre = </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sumSorguLBL">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Toplam yapılan sorgu adeti = </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="noktalarCheckBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>noktalar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="topPointCheckBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>en yüksek nokta</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="bottomPointCheckBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>en düşük nokta</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="saveKmlBtn">
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>220</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>noktaları kaydet</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="savedKmlLinkLBL">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>:::...</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
